lib/modules/manager/cpanfile/parser.ts

```diff
@@ -0,0 +1,127 @@
+import { query as q } from 'good-enough-parser';
+import { CpanDatasource } from '../../datasource/cpan';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import * as perlVersioning from '../../versioning/perl';
+import type { PackageDependency } from '../types';
+import { cpanfile } from './language';
+
+interface Ctx {
+  deps: PackageDependency[];
+
+  perlVersion?: string;
+
+  phase?: string;
+  tempPhase?: string;
+
+  depName?: string;
+  currentValue?: string;
+}
+
+const perlVersionMatch = q
```

Comment:
Please comment the sample construct being parsed by every of matchers like this one (like it's done for `gradle`)